lib/Runtime/Library/SIMDUint8x16Lib.cpp

```diff
@@ -265,7 +264,7 @@ namespace Js
 
         if (args.Info.Count >= 4 && JavascriptSIMDUint8x16::Is(args[3]))
         {
-            SIMD128TypedArrayStore<JavascriptSIMDUint8x16>(args[1], args[2], args[3], 16 * TySize[TyUint8], scriptContext);
+            SIMD128TypedArrayStore<JavascriptSIMDUint8x16>(args[1], args[2], args[3], 16 * INT8_SIZE, scriptContext);
```

Comment:
As a general comment, I find it a little confusing to confirm this because the type `JavascriptSIMDUint8x16` looks like it's saying 8 of 16-bit values, and the math seems to have that reversed.